### ISSUE TITLE
Start up Command

### ISSUE THREAD

**Issue by Akinnagbe** [Azure-App-Service/ruby#35]
Hello, 

I added a start up command to run migrations while the app is starting up using the Start up command section in Configuration=>General Settings tab. but i noticed that the container fails to start showing the following error:

2019-09-04 14:35:14.810 ERROR - Container ****** for site ..... has exited, failing site start
2019-09-04 14:35:14.825 ERROR - Container ****** didn't respond to HTTP pings on port: 8080, failing site start. See container logs for debugging.

**Comment by dombarnes** [comment]
Does the startup command work outside of docker? I assume its something like
`bundle exec rake db:migrate && bundle exec rails`

**Comment by Akinnagbe** [comment]
yes it works, it was initially hosted on Heroku without issues, but when ported to Azure, the start up commands didn't work.

**Comment by pjmartorell** [comment]
@Akinnagbe I think your error is related with the character encoding of your script. Check this article: https://omgdebugging.com/2017/12/22/azure-web-app-for-container-failing-site-start/

Another option is to try `rbenv exec bundle exec rake db:migrate`
